File: app/src/test/java/com/tomaszrykala/dorefindmi/things/suppliers/buzzer/MockBuzzerSupplier.java
package com.tomaszrykala.dorefindmi.things.suppliers.buzzer;

public class MockBuzzerSupplier implements BuzzerSupplier {

    @Override public void play(double pitch) {
        System.out.print("MockBuzzerSupplier.play; ");
        System.out.println("pitch = [" + pitch + "]");
    }

    @Override public void close() throws Exception {
        System.out.println("MockBuzzerSupplier.close");
    }
}

File: app/src/test/java/com/tomaszrykala/dorefindmi/things/supplier/buzzer/MockBuzzerSupplier.kt
package com.tomaszrykala.dorefindmi.things.supplier.buzzer

class MockBuzzerSupplier : BuzzerSupplier {

    override fun play(pitch: Double) {
        print("MockBuzzerSupplier.play; ")
        println("pitch = [$pitch]")
    }

    @Throws(Exception::class)
    override fun close() {
        println("MockBuzzerSupplier.close")
    }
}
